release-1.7/release-1.7.md

```diff
@@ -0,0 +1,58 @@
+# Proposed timeline for 1.7
+We will begin the 1.7 release the first week of April 2017.
+
+The proposal below is identical in layout to the 1.5 and 1.6 plans with some
+modifications:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+
+## 1.7 Release Schedule
+- April 3 (Mon) coding start (7w)
+- April 26 (Weds) features repo freeze
+- May 17th (Weds) v1.7.0-alpha.1
```

Comment:
Looks like this is in flight.  I'll add the upcoming alpha releases to the schedule (every 2 weeks starting yesterday)